diff --git a/dist/adapters/node-http/core.d.ts b/dist/adapters/node-http/core.d.ts
index 7a21316dd709af70fa4e0d140b5d3e7babb41a76..0e97cbf9f5d5c1a56e7218e4ffeed06444c901f2 100644
--- a/dist/adapters/node-http/core.d.ts
+++ b/dist/adapters/node-http/core.d.ts
@@ -1,4 +1,4 @@
-import { NodeHTTPHandlerOptions, NodeHTTPRequest, NodeHTTPResponse } from '@trpc/server/dist/adapters/node-http';
+import { NodeHTTPHandlerOptions, NodeHTTPRequest, NodeHTTPResponse } from '@trpc/server/adapters/node-http';
 import { OpenApiRouter } from '../../types';
 export type CreateOpenApiNodeHttpHandlerOptions<TRouter extends OpenApiRouter, TRequest extends NodeHTTPRequest, TResponse extends NodeHTTPResponse> = Pick<NodeHTTPHandlerOptions<TRouter, TRequest, TResponse>, 'router' | 'createContext' | 'responseMeta' | 'onError' | 'maxBodySize'>;
 export type OpenApiNextFunction = () => void;
